### PR TITLE
Create stcs_regions

### PR DESCRIPTION
**Pull request by keflavich**
cc @cdeil @migueldvb - do either of you have time to help advise such a project?  It would be really great to have this built, and I think based off last year's CRTF project, it's perfect scale for a summer project.

### CODE REVIEW

**Comment by keflavich** [comment]
This might actually be a lot smaller, as in, it might just be "port this https://github.com/at88mph/opencadc_stc/tree/master/opencadc_stc into astropy-regions"

File: _projects/2019/astropy/stcs_regions [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Different technologies needed
+ - python
+ - regions
+ - astropy
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - astropy
+ - regions
+---
+Implement the STC-S region type for astropy
```
Comment (dpshelio) [inline _projects/2019/astropy/stcs_regions]: I would remove this line and named the file as `.md`
```suggestion

```

**Comment by cdeil** [comment]
>  @cdeil @migueldvb - do either of you have time to help advise such a project?

I don't have time, unfortunately. Good luck!

**Comment by migueldvb** [comment]
I cannot make it unfortunately! I think that the opencadc_stc implementation is the right approach, however the code is released under GPLv3 which may be incompatible with regions' BSD license?

**Comment by bsipocz** [comment]
I'm merging this now with the common understanding that the project needs one more mentor to be able to run, that I'm positive we are able to solve within astropy in case we get a good proposal for the project.

**Pull request merged**